src/main/java/bio/terra/service/resourcemanagement/google/GoogleBillingService.java

```diff
@@ -73,38 +91,32 @@ private static Cloudbilling cloudbilling() {
     @Override
     public boolean canAccess(BillingProfile billingProfile) {
         String accountId = "billingAccounts/" + billingProfile.getBillingAccountId();
+        ResourceName resource = BillingAccountName.of(accountId);
         List<String> permissions = Collections.singletonList("billing.resourceAssociations.create");
-        TestIamPermissionsRequest permissionsRequest = new TestIamPermissionsRequest().setPermissions(permissions);
+        TestIamPermissionsRequest permissionsRequest = TestIamPermissionsRequest.newBuilder()
+            .setResource(resource.toString())
```

Comment:
It seems like this is the new suggested method to test IAM permissions for a billing account.